assets/dev/scss/frontend/widgets/icon-box.scss

```diff
@@ -1,48 +1,90 @@
 //
 // Icon Box
 //
-.elementor-widget-icon-box {
 
-	@media (min-width: $screen-mobile-next) {
+@mixin icon-position($device) {
+	&.elementor#{$device}-position- {
 
-		&.elementor-position- {
+		&left,
+		&right {
 
-			&left,
-			&right {
+			.elementor-icon-box-wrapper {
+				display: flex;
+			}
 
-				.elementor-icon-box-wrapper {
-					display: flex;
-				}
+			.elementor-icon-box-icon {
+				display: inline-flex;
+				flex: 0 0 auto; // Hack for Safari
+			}
 
-				.elementor-icon-box-icon {
-					display: inline-flex;
-					flex: 0 0 auto; // Hack for Safari
-				}
+		}
+
+		&right {
+
+			.elementor-icon-box-wrapper {
+				text-align: $end;
+				flex-direction: getValueByDirection(row-reverse, row);
 			}
 
-			&right {
+			.elementor-icon-box-icon {
+				margin-left: var(--icon-box-icon-margin);
+				margin-right: 0;
+				margin-bottom: unset;
+			}
 
-				.elementor-icon-box-wrapper {
-					text-align: $end;
-					flex-direction: getValueByDirection(row-reverse, row);
-				}
+		}
+
+		&left {
+
+			.elementor-icon-box-wrapper {
+				text-align: $start;
+				flex-direction: getValueByDirection(row, row-reverse);
 			}
 
-			&left {
+			.elementor-icon-box-icon {
+				margin-right: var(--icon-box-icon-margin);
+				margin-left: 0;
+				margin-bottom: unset;
+			}
 
-				.elementor-icon-box-wrapper {
-					text-align: $start;
-					flex-direction: getValueByDirection(row, row-reverse);
-				}
+		}
+
+		&top {
+
+			.elementor-icon-box-wrapper {
+				display: block;
+				text-align: center;
+				flex-direction: unset;
 			}
 
-			&top {
+			.elementor-icon-box-img {
```

Comment:
What is this class?

Reply:
You mean this class ".elementor-icon-box-img"?
I am actually not sure. 
If copied and pasted it from the original version.